client/css/style.css

```diff
@@ -0,0 +1,235 @@
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#uploadControls {
+    border: var(--light-muted) 1px solid;
+    padding: 8px;
+    border-radius: 4px;
+    background-color: var(--light-slightly-muted);
+}
+
+#query {
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: column;
+    justify-content: flex-start;
+    border: var(--dark-muted) 1px solid;
+    margin-left: 8px;
+    border-radius: 8px;
+    padding: 12px;
+}
+
+#response {
+    margin: 12px 0px;
+}
+
+input[type="text"],textarea {
+    border-radius: 4px;
+    font-size: 1.2em;
+    width: 100%;
+}
+
+textarea {
+    height: 70px;
+}
+#recordings {
+    display: flex;
+    flex-direction: column;
+}
+
+#manual {
+    border: var(--dark-muted) 1px solid;
+    width: 100%;
+    border-radius: 8px;
+    padding: 12px;
+}
+
+#ask {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+#uploadError {
+    color: red;
+    font-size: 0.8em;
+}
+
+
+.lobby {
+    display: flex;
+    flex-direction: column;
+    width: 70%
+}
+
+.lobby a {
+    color: var(--orange-dark);
+    text-decoration: none;
+}
+
+.lobby a:hover:enabled {
+    text-decoration: underline;
+}
+
+
+
+/* Form and button styling */
+
+form {
+    margin-bottom: 8px;
+    margin-bottom: 8px;
+    display: flex;
+    flex-direction: column;
+}
+
+form button {
+    font-size: 1.2em;
+    margin: 4px;
+}
+
+.dark-btn {
+    background-color: var(--dark);
+    border-color: var(--dark);
+    border-radius: 24px;
+    border-width: 1px;
+    color: var(--light);
+    transition: 0.5s;
+    padding: 8px;
+}
+
+.dark-btn:hover:enabled {
+    background-color: var(--light);
+    color: var(--dark);
+    cursor: pointer;
+}
+
+.light-btn:not([type='file']),.light-btn::file-selector-button {
+    background-color: var(--light);
+    border-color: var(--dark);
+    border-radius: 28px;
+    border-width: 1px;
+    color: var(--dark);
+    transition: 0.5s;
+    padding: 8px;
+}
+
+.light-btn:not([type='file']):hover:enabled,.light-btn::file-selector-button:hover {
+    background-color: var(--dark);
+    color: var(--light);
+    cursor: pointer;
+}
+
+.teal-btn,.teal-btn::file-selector-button {
+    background-color: var(--teal);
+    border-color: var(--dark);
+    border-radius: 28px;
+    border-width: 1px;
+    color: var(--dark);
+    transition: 0.5s;
+    padding: 8px;
+    font-size: 1.1em;
+}
+
+.teal-btn:hover:enabled,.teal-btn::file-selector-button:hover {
+    background-color: var(--light);
+    cursor: pointer;
+}
+
+
+/* Spinner */
+.spinner {
+    border: 4px solid var(--light); 
+    border-top: 4px solid var(--teal);
+    border-radius: 50%;
+    width: 8px;
+    height: 8px;
+    animation: spin 1s linear infinite;
+    margin-left: 4px;
+}
+  
+@keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+}
+
+.hidden {
+    display: none;
+}
+
+.disabled {
+    background-color: var(--dark-muted);
```

Comment:
Nit: I would make this a bit lighter (the background color) -- either #737374 (or whatever you colour var equivalent is) or opacity:0.6. And then add a cursor to indicate it can't be clicked
```suggestion
    background-color: var(--dark-muted);
    cursor: not-allowed;
```